lib/Parser/MimeDir.php

```diff
@@ -215,12 +215,20 @@ protected function parseLine($line)
                         && $e instanceof ParseException && str_contains($e->getMessage(), 'Invalid Mimedir file. Line starting at')
                         && ($this->options & Reader::OPTION_FIX_UNFOLDING)
                     ) {
-                        // Fix unfolding
-                        $component->remove($prevNode);
-                        $value = $prevNode->getValue() . ' ' . $line . PHP_EOL;
-                        $prevNode->offsetSet('VALUE', $value);
-                        $prevNode->setValue($value);
-                        $component->add($prevNode);
+                        if (preg_match('/[A-Z]\w+; /', $line)) {
+                            $line = preg_replace('/([A-Z]\w+;) /', '$1', $line);
+                            $result = $this->parseLine($line);
+                            if ($result) {
+                                $prevNode = $component->add($result);
+                            }
```

Comment:
Can this have false positives? What happens when parseLine fails?

Reply:
It will throw the exception, for example with 2 empty space. Not sure if we should cover it or let it fails